mypy/checkmember.py

```diff
@@ -1400,6 +1401,67 @@ def analyze_typeddict_access(
             fallback=mx.chk.named_type("builtins.function"),
             name=name,
         )
+    elif name == "get":
```

Comment:
This whole synthesized signature should probably be made part of the TypeInfo, but I am not sure where exactly to put the code.